content/2_Bronze/Intro_Sets.mdx

```diff
@@ -755,3 +755,203 @@ Some of these problems can be solved by sorting alone, though sets or maps could
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $-1$
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $-3$
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+	```cpp
+	#include <bits/stdc++.h>
+	using namespace std;
+	int main()
+	{
+		vector<int> a = {4, 1, 2, 1, 3, 2, 3, 1, 3, 1};
+		set<int> s;
+		for(int x: a) s.insert(x);
+		cout << s.size() << endl;
+	}
+```
+    <Quiz.Answer>
+      $1$
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $4$
+      <Quiz.Explanation>
+        There are a total of $4$ unique numbers inserted into $s$, so the answer is $4$.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $7$
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $10$
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What is the time complexity of insertions, deletions, and removals in an ordered map?
+    <Quiz.Answer>
+      $\mathcal{O}(1)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N \log N)$
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $\mathcal{O}(\log N)$
+      <Quiz.Explanation>
+				Correct.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+	```cpp
+	#include <bits/stdc++.h>
+	using namespace std;
+	int main()
+	{
+		map<int, int> m;
+		m[1] = 5;
+		m[2] = 9;
+		m[3] = 7;
+		m[4] = 3;
+		cout << m[2] << endl;
+	}
+	```
+    <Quiz.Answer>
+      $5$
+      <Quiz.Explanation>
+	      
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $9$
+      <Quiz.Explanation>
+        The value associated with the key $2$ is $9$.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $2$
+      <Quiz.Explanation>
+	      
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $0$
+      <Quiz.Explanation>
+	      
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+	```cpp
+	#include <bits/stdc++.h>
+	using namespace std;
+	int main()
+	{
+		map<int, int> m;
+		m[1] = 5;
+		m[2] = 9;
+		m[3] = 7;
+		m[4] = 3;
+		m.erase(2);
+		cout << m[2] << endl;
+	}
+    ```
+    <Quiz.Answer correct>
+      $0$
+      <Quiz.Explanation>
+        After ```m.erase(2)``` is called, there is no longer a value associated with the key $2$, meaning ```m[2]``` is $0$.
```

Comment:
do you think it would be helpful to reference [this](https://stackoverflow.com/questions/10124679/what-happens-if-i-read-a-maps-value-where-the-key-does-not-exist) to clarify why it's 0 in this case, since this could vary.

Reply:
Well it says it's 0 in the module so idt it's necessary?

Reply:
@maggie-j-liu said that the shit should be about the stuff in the module

Reply:
ok, can you add that to the module then (since it's not necessarily always 0)

Reply:
Uh can you put it there cuz i don't rly understand this.

Reply:
don't resolve this until it's been fixed